legoify.py

```diff
@@ -8,13 +8,13 @@
 brickw = 30
 brick = "brick30.png"
 
-cmdargs = str(sys.argv)
+cmdargs = sys.argv[1]
```

Comment:
this sends an IndexError when there is not argument provided, I'll make a commit